### PR TITLE
Adjusted the webpage layout and did some tweaking with the mobile version.

### PR DESCRIPTION
**Pull request by AnshumanMahato**
Made some changes in the navbar and nav menu (mobile version). Did some minor layout tweaking as well. Visit this [link](https://anshumanmahato.github.io/web/) to see those changes.

### CODE REVIEW

**Review by castdrian — APPROVED** [review]
Looking good to me

File: about.html [hunk 1]
```diff
@@ -24,13 +24,17 @@
         </div>
         <nav>
             <div class="left">
-                <a id="menu" class="hide"><i class="fas fa-bars"></i>  MENU</a>
-                <a href="index.html" class="show">HOME</a>
-                <a href="about.html" class="show selected">ABOUT</a>
+                <a href="index.html" class="show selected">HOME</a>
```
Comment (MBR-0001) [inline about.html]: The `selected` class was intentionally set to different item on every page for [this reason](https://cdn.mbr.pw/tKUcvSY.mp4) (accessibility)

File: css/style.css [hunk 2]
```diff
@@ -94,9 +114,10 @@ nav {
     transition: background-color 0.2s;
     font-size: 16px;
     font-weight: 600;
+    margin: 30px 10px 0;
```
Comment (MBR-0001) [inline css/style.css]: although I like the change the margins are a little too big IMO (mobile version)

File: css/style.css [hunk 3]
```diff
@@ -153,7 +176,6 @@ a { color: red; }
 @media (max-width: 768px) {
     .hide {
         display: block;
-        width: 100%;
```
Comment (MBR-0001) [inline css/style.css:156]: why was this removed? this ensured the buttons can be clicked anywhere (on mobile) instead only where the text is ([picture](https://cdn.mbr.pw/9YZaNB7.png)) (also accessibility)

File: css/style.css [hunk 4]
```diff
@@ -1,13 +1,17 @@
-* { font-family: Poppins; }
+* { 
+    font-family: Poppins; 
+    box-sizing: border-box;
```
Comment (MBR-0001) [inline css/style.css:3]: this kinda broke the features page ([image](https://cdn.mbr.pw/NPmUBsr.png), [before](https://cdn.mbr.pw/7zh2HtW.png)), could be easily fixed by setting it back to block for .card

**Review by MBR-0001 — CHANGES_REQUESTED** [review]

File: css/style.css (hunk 2 above)
Comment (AnshumanMahato) [reply css/style.css]: How much margin do you suggest for this?

Comment (MBR-0001) [reply css/style.css]: from testing `10px 5px 0` looks good on both mobile & desktop view

File: about.html (hunk 1 above)
Comment (AnshumanMahato) [reply about.html]: The `selected` class has been added to respective anchor tags following the pages.

File: css/style.css (hunk 2 above)
Comment (AnshumanMahato) [reply css/style.css]: margins adjusted to '10px 5px 0'.

File: css/style.css (hunk 3 above)
Comment (AnshumanMahato) [reply css/style.css:156]: That was removed for bringing the menu icon inline with the Gideon text in the navbar for mobile screens. Resolution on line 193 in styles.css

File: css/style.css (hunk 4 above)
Comment (AnshumanMahato) [reply css/style.css:3]: Resolution added on line 16,17 in features.css

**Review by MBR-0001 — APPROVED** [review]
